JobConfig/beam/MuBeamSplitter.fcl

```diff
@@ -0,0 +1,24 @@
+#
+# split the Muon Beam stream into muons and antimuons
+# This is run after the beam stage
+#
+#include "Production/JobConfig/beam/BeamSplitter.fcl"
+process_name: mubeamsplitter
+
+outputs: {
+  muplusout: {
+    module_type: RootOutput
+    fileName : "sim.owner.MuPlusBeamCat.version.sequencer.art"
+    outputCommands:   [ "keep *_*_*_*" ]
+    SelectEvents: [ selectMuons ]
+  }
+  muminusout: {
+    module_type: RootOutput
+    fileName : "sim.owner.MuMinusBeamCat.version.sequencer.art"
+    outputCommands:   [ "keep *_*_*_*" ]
+    RejectEvents: [ selectMuons ]
+  }
+}
+
+physics.filters.muonSelector.ParticleCodes : [ [-13, "uninitialized" , "mu2eKillerVolume" ] ]
+physics.outpath: [ muplusout, muminusout ]
```

Comment:
Can you exclude the muminusout from the path, since we don't need that?

Reply:
We don't need it now but maybe it can be useful in the future? If this is about the production I can just not copy back the muminus files

Reply:
Ok I removed it